### PR TITLE
Logout function. Wipe login Data

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

Logout now wipe login data of the Local Storage and Redux.

IMAGE ALT TEXT HERE

## Trello Card
https://trello.com/c/Uopsgma5/77-logout-implementaci%C3%B3n

### CODE REVIEW

File: src/localStorage/index.tsx [hunk 1]
```diff
@@ -26,3 +26,8 @@ export const getLoginData = async () => {
 
   return data;
 };
+
+export const wipeLoginData = async () => {
+  const keys = ['@accessToken', '@client', '@uid'];
```
Comment (guidoprinc) [inline src/localStorage/index.tsx:31]: use same constants as the other PR

Comment (guidoprinc) [reply src/localStorage/index.tsx:31]: #13

Comment (mcolombo87) [reply src/localStorage/index.tsx]: Done

File: src/app/components/LogoutHeader/index.tsx [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+import actionCreators from '@redux/login/actions';
 import styles from './styles';
 
 const LogoutHeader = () => {
+  const navigation = useNavigation();
+  const dispatch = useDispatch();
+  const logOut = () => {
+    wipeLoginData();
+    dispatch(actionCreators.clearLoginData());
+    navigation.navigate('Login');
```
Comment (guidoprinc) [inline src/app/components/LogoutHeader/index.tsx:15]: you should reset navigation, not navigate because you can go back on navigate actions

Comment (mcolombo87) [reply src/app/components/LogoutHeader/index.tsx]: Done!

Comment (mcolombo87) [reply src/app/components/LogoutHeader/index.tsx]: (Anyway, I can tell ... when I logged out, the Back button does not navigate to another view)

**Pull request merged**